lib/ros/be/infra/templates/terraform/aws/kubernetes.tf.erb

```diff
@@ -62,7 +62,7 @@ locals {
   accelerator_name                    = "<%= infra.components.globalaccelerator&.config&.name.nil? ? infra.config.cluster.name : infra.components.globalaccelerator.config.name %>"
   iam_name                            = "<%= infra.components.iam&.config&.name.nil? ? infra.config.cluster.name : infra.components.iam.config.name %>"
   tags                                = var.tags
-  domain_name                         = "<%= infra.dns.sub_domain %><%= infra.dns.sub_domain.nil? ? "" : "."%><%= infra.dns.root_domain %>"
+  domain_name                         = "<%= infra.config.cluster.dns.subdomain %><%= infra.config.cluster.dns.subdomain.nil? ? "" : "."%><%= infra.config.cluster.dns.domain %>"
```

Comment:
I think you should not change this. the config files `production.yaml`, `production-uat.yaml` and `production-staging.yaml` set the DNS section at a different place.

We can remove our `eks-develop.yaml` config file.

Reply:
You are right...confused between multiple configs...will push an update shortly